Readme.md

```diff
@@ -15,9 +15,15 @@ $ npm install tempdir --save
 ```javascript
 const tempdir = require('tempdir');
 
-let path = tempdir();
+tempdir().then(
```

Comment:
Sure, I mean it's your repo, you can do what you want with it =). Personally I think it's worth pointing out how to handle errors, but I suppose anyone familiar with promises should already know that.